odoo/addons/base/rng/common.rng

```diff
@@ -260,6 +260,7 @@
             <rng:optional><rng:attribute name="filters"/></rng:optional>
             <rng:optional><rng:attribute name="statusbar_visible"/></rng:optional>
             <rng:optional><rng:attribute name="can_create" /></rng:optional>
+            <rng:optional><rng:attribute name="allow_drag_drop_create"/></rng:optional>
```

Comment:
no sure about the option name, nor if we want to mix two features into one option

Reply:
Conversation conclusion : it has been decided to revert to the previous denomination `can_use_group_value` and restrict its usage to only `date` and `datetime` fields. The default value when `undefined` will be `false`

Reply:
I respectfully don't agree as both Awa and me found it clearer :D . Do you really want to start a war ? :crossed_swords: